data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -305,26 +310,75 @@
         [/message]
 
         [message]
             speaker=narrator
-            caption= _ "Abilities and Weapon Specials"
+            caption= _ "Weapon Specials"
             image=wesnoth-icon.png
-            message= _ "Weapon specials are special attack modifiers that make certain attacks more powerful. Once again, in this case, the Shaman has the Slows special on her entangle attack. When this attack is used on an enemy, it will halve the damage they do for one turn. Other weapon specials include First Strike, which allows that attack to always hit first no matter who attacks first, and Magical, which gives that attack a constant 70% chance to hit.
+            message= _ "Weapon specials are special attack modifiers that make certain attacks more powerful. In this case, the Shaman has the Slows special on her entangle attack. When this attack successfully hits an enemy, it will halve the damage they do for one turn. Other weapon specials include First Strike, which allows that attack to always hit first no matter who attacks first, and Magical, which gives that attack a constant 70% chance to hit.
 
 A full list of abilities and weapons specials, along with traits, may be found in the help (default hotkey: <b>F1</b>)."
         [/message]
     [/event]
 
+    # Triggers the first time that the player has two healers.
+    # If there are allied sides then this isn't accurate, but at this stage of the tutorial allied sides haven't been introduced yet.
+    [event]
+        name=recruit
+        [filter]
+            side=1
+            type=Elvish Shaman
+        [/filter]
+        [filter_condition]
+            [have_unit]
+                ability_type=heals
+                count=2
+            [/have_unit]
+        [/filter_condition]
+
+        [if]
+            [have_unit]
+                type=Elvish Shaman
+                count=2
+            [/have_unit]
+            [then]
+                [message]
+                    speaker=unit
+                    # po: the speaker and the other healer are both level one elvish shamans
+                    message= _ "You now have two healers. However, each unit can only receive healing from one healer — even if we’re both next to a unit then that unit will still only be healed 4 hitpoints per turn.
+
+If either of us advances to become a Druid, she’ll be able to heal adjacent units by 8 hitpoints per turn. A unit next to two healers is healed by the more advanced of the two."
+                [/message]
+            [/then]
+            [else]
+                [message]
+                    speaker=unit
+                    # po: the speaker is a shaman, but the other healer is a druid or (very unlikely) a shyde
+                    message= _ "You now have two healers. Your Druid can heal adjacent units by 8 hitpoints per turn compared to my 4. However, each unit can only receive healing from one healer — a unit next to two healers is healed by the more advanced of the two."
```

Comment:
```suggestion
                    message= _ "Shaman reporting in! I can heal adjacent units by 4 hitpoints per turn compared to my Druid friend’s 8. However, only one of us can work at a time — a unit next to multiple healers is healed by only the most advanced one."
```

Reply:
I'll take the change to multiple, but not the "reporting in" nor the druid friend.